source/library/buttercup.js

```diff
@@ -98,7 +99,8 @@ export function getArchiveEncryptedContent(archive, credentials) {
 
 export function getSharedArchiveManager() {
     if (__sharedManager === null) {
-        __sharedManager = new ArchiveManager(new AsyncStorageInterface());
+        // __sharedManager = new ArchiveManager(new AsyncStorageInterface());
```

Comment:
I guess this can be removed?

Reply:
Removed in 908fdcf1e8aa10f5c716fe73d028dbc6ed26d5e9